content/intro-to-storybook/react/it/get-started.md

```diff
@@ -1,74 +1,74 @@
 ---
-title: 'Storybook for React tutorial'
-tocTitle: 'Get started'
-description: 'Set up Storybook in your development environment'
+title: 'Tutorial di Storybook per React'
+tocTitle: 'Inizia'
+description: 'Configura Storybook nel tuo ambiente di sviluppo'
 commit: 'b3cfa66'
 ---
 
-Storybook runs alongside your app in development mode. It helps you build UI components isolated from the business logic and context of your app. This edition of the Intro to Storybook tutorial is for React; other editions exist for [React Native](/intro-to-storybook/react-native/en/get-started), [Vue](/intro-to-storybook/vue/en/get-started), [Angular](/intro-to-storybook/angular/en/get-started), [Svelte](/intro-to-storybook/svelte/en/get-started) and [Ember](/intro-to-storybook/ember/en/get-started).
+Storybook funziona in parallelo con la tua app in modalità di sviluppo. Ti aiuta a costruire componenti dell'interfaccia utente isolati dalla logica aziendale e dal contesto della tua app. Questa edizione del tutorial Intro to Storybook è per React; esistono altre edizioni per [React Native](/intro-to-storybook/react-native/it/get-started), [Vue](/intro-to-storybook/vue/en/get-started), [Angular](/intro-to-storybook/angular/en/get-started), [Svelte](/intro-to-storybook/svelte/en/get-started) ed [Ember](/intro-to-storybook/ember/en/get-started).
 
-![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
+![Storybook e la tua app](/intro-to-storybook/storybook-relationship.jpg)
 
-## Set up React Storybook
+## Configura Storybook per React
 
-We'll need to follow a few steps to get the build process set up in our environment. To start with, we want to use [degit](https://github.com/Rich-Harris/degit) to set up our build system. Using this package, you can download "templates" (partially built applications with some default configuration) to help you fast track your development workflow.
+Dovremo seguire alcuni passi per configurare il processo di build nel nostro ambiente. Per iniziare, vogliamo utilizzare [degit](https://github.com/Rich-Harris/degit) per configurare il nostro sistema di build. Utilizzando questo pacchetto, puoi scaricare "modelli" (applicazioni parzialmente costruite con alcune configurazioni predefinite) per aiutarti a velocizzare il tuo flusso di lavoro di sviluppo.
```

Comment:
**puoi scaricare "modelli"** it can be changed in **puoi scaricare dei "modelli"** to make it more readable.